### PR TITLE
Add a new test send_hello_txes()

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: coordinator/src/lib.rs [hunk 1]
```diff
@@ -116,31 +116,32 @@ impl Coordinator {
...
+                .unwrap_or_else(|| {
+                    sessions.push(0);
+                    (sessions.len() - 1, 0)
+                });
 
-        sessions[index] |= 1 << bit;
-        let session_id = bit + (SESSION_BITS_PER_SLOT * index) as SessionId;
+            sessions[index] |= 1 << bit;
+            bit + (SESSION_BITS_PER_SLOT * index) as SessionId
+        };
```
Comment (majecty) [inline coordinator/src/lib.rs:138]: @junha1 Could you explain how session is using bitset?

File: integration-test/tests/integration_test.rs [hunk 2]
```diff
@@ -91,3 +91,39 @@ async fn send_hello_tx() {
     let value: Value = serde_json::from_str(&query_result).unwrap();
     assert_eq!(value["data"]["account"]["seq"], 1);
 }
+
+#[actix_rt::test]
+async fn send_hello_txes() {
```
Comment (majecty) [inline integration-test/tests/integration_test.rs:96]: How about making the test name to express the purpose?
This test is checking the sequence management code. Could you update the name of the test?

**Review by majecty — CHANGES_REQUESTED** [review]

File: coordinator/src/lib.rs (hunk 1 above)
Comment (junha1) [reply coordinator/src/lib.rs:138]: @dynaxis authored this.

Comment (dynaxis) [reply coordinator/src/lib.rs:138]: It's a bit more complex than absolutely required. I want to assign the smallest integer ID available and manage unoccupied IDs by mapping them to 0 bits (naturally 1 bits for occupied IDs). The code can be implemented using `bool` vector instead.

File: integration-test/tests/integration_test.rs (hunk 2 above)
Comment (junha1) [reply integration-test/tests/integration_test.rs:96]: @majecty Would you recommend me a nice name?

Comment (majecty) [reply integration-test/tests/integration_test.rs:96]: How about `sequence_management`?
We may make tests like `sequence_management_1`, `sequence_management_2`, ...

Comment (junha1) [reply integration-test/tests/integration_test.rs:96]: Ok. I'll update

Comment (junha1) [reply integration-test/tests/integration_test.rs:96]: ok

File: timestamp/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Timestamp Modules
```
Comment (majecty) [inline timestamp/README.md:1]: Could you create a new commit for this README.md file?
It is contained in " Force single thread for GraphQL tokio runtime " commit.

**Review by majecty — CHANGES_REQUESTED** [review]

File: timestamp/README.md (hunk 3 above)
Comment (junha1) [reply timestamp/README.md:1]: OMG. My mistake. I'll remove it

**Review by majecty — APPROVED** [review]

**Pull request merged**